package.json

```diff
@@ -8,7 +8,6 @@
     "test": "tests"
   },
   "scripts": {
-    "db:seed": "cd services/app-api && yarn run db:seed",
```

Comment:
No one uses this?

Reply:
Nope not yet. In other repos we have seed files and seed lambdas but that hasn't been necessary for HCBS yet.